src/openhmd.c

```diff
@@ -97,6 +98,18 @@ int OHMD_APIENTRY ohmd_ctx_probe(ohmd_context* ctx)
 	return ctx->list.num_devices;
 }
 
+const char* OHMD_APIENTRY ohmd_gets(ohmd_string_description type)
```

Comment:
Sorry for the delay. We should probably have it return an int. ohmd_list_gets() should've too, but it's an old function.

so `int OHMD_APIENTRY ohmd_gets(ohmd_string_description type, const char** str);`